main.js

```diff
@@ -31,7 +32,7 @@ if(program.validate) {
 
 SwaggerParser.bundle(apiFilePath)
   .then(function(api) {
-    openapi2slate.printSlateMarkdown(api, program.includeInternal);
+    openapi2slate.printSlateMarkdown(api, program.includeInternal, Boolean(program.searchFilter));
```

Comment:
The explicit bool cast shouldn't be necessary. Previous argument is also bool and doesn't use it.

Reply:
That's what I thought as well, but it looks like `program.searchFilter` is `undefined` when the `--search-filter` parameter is not provided.  It functions as a "falsy" value but gets printed as "undefined", which leads to `search: undefined` on output of the command.  An explicit cast to Boolean solves the problem.